content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -429,8 +432,8 @@ The following table lists configuration variables that are available only when u
 
 {{< partial name="whats-next/whats-next.html" >}}
 
-[1]: /tracing/compatibility_requirements/dotnet-core
-[2]: https://app.datadoghq.com/apm/docs
+[1]: https://app.datadoghq.com/apm/docs
+[2]: /tracing/compatibility_requirements/dotnet-core
```

Comment:
```suggestion
[1]: /tracing/setup_overview/compatibility_requirements/dotnet-framework
[2]: /tracing/setup_overview/compatibility_requirements/dotnet-core
```

Reply:
on the dotnet-core page we should only be linking to [2], so all [1]'s and [2]'s should become [2] and link to dotnet-core